backend-es/src/Main.purs

```diff
@@ -377,24 +381,26 @@ esModulePackageJson = """{"type": "module"}"""
 
 spawnFromParentWithStdin :: String -> Array String -> Maybe String -> Aff Unit
 spawnFromParentWithStdin command args input = makeAff \k -> do
-  childProc <- ChildProcess.spawn command args defaultSpawnOptions
-    { stdio =
-        [ Just $ Pipe
-        , Just $ ShareStream (unsafeCoerce Process.stdout)
-        , Just $ ShareStream (unsafeCoerce Process.stderr)
+  -- To preserve colors in stdout need to pass it into spawn's stdio.
+  childProc <- unsafeCoerce <$> UnsafeChildProcess.spawn' command args
+    { stdio:
+        [ pipe
+        , shareStream Process.stdout
+        , shareStream Process.stderr
         ]
+    , shell: enableShell
     }
   for_ input \inp -> do
-    _ <- Stream.writeString (ChildProcess.stdin childProc) UTF8 inp mempty
-    Stream.end (ChildProcess.stdin childProc) mempty
-  ChildProcess.onExit childProc case _ of
+    _ <- Stream.writeString (ChildProcess.stdin childProc) UTF8 inp
+    Stream.end (ChildProcess.stdin childProc)
+  childProc # on_ exitH case _ of
     Normally code
-      | code > 0 -> Process.exit code
+      | code > 0 -> Process.exit' code
       | otherwise -> k (Right unit)
     BySignal _ ->
-      Process.exit 1
+      Process.exit' 1
   pure $ effectCanceler do
-    ChildProcess.kill SIGABRT childProc
+    void $ ChildProcess.kill' (stringSignal "SIGABRT") childProc
```

Comment:
Can this use https://pursuit.purescript.org/packages/purescript-node-child-process/11.1.0/docs/Node.ChildProcess#v:killSignal to avoid the string? These functions are largely undocumented so I'm not really sure what the difference is.

Reply:
Missed killSignal.